### PR TITLE
Ad 185421 v2

### PR DESCRIPTION
**Pull request by amjones99**

### CODE REVIEW

File: NCS.DSS.Interaction/APIDefinition/APIDefinition.cs
Comment (bmatthews-DFE) [inline NCS.DSS.Interaction/APIDefinition/APIDefinition.cs:1]: Don't forget to increment the API version #

File: NCS.DSS.Interaction/NCS.DSS.Interaction.csproj [hunk 1]
```diff
@@ -1,24 +1,29 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-	  <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <TargetFramework>net8.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>enabled</ImplicitUsings>
```
Comment (bmatthews-DFE) [inline NCS.DSS.Interaction/NCS.DSS.Interaction.csproj:6]: Replace `<ImplicitUsings>enabled</ImplicitUsings>` with `<ImplicitUsings>enable</ImplicitUsings>`

**Review by bmatthews-DFE — COMMENTED** [review]
This build appears to have obselete warnings;
image

File: NCS.DSS.Interaction/PostInteractionHttpTrigger/Function/PostInteractionHttpTrigger.cs
Comment (PradeepVoorukonda) [inline NCS.DSS.Interaction/PostInteractionHttpTrigger/Function/PostInteractionHttpTrigger.cs:1]: _httpResponseMessageHelper and jsonHelper not used any more. These references can be removed

File: NCS.DSS.Interaction/PatchInteractionHttpTrigger/Function/PatchInteractionHttpTrigger.cs
Comment (PradeepVoorukonda) [inline NCS.DSS.Interaction/PatchInteractionHttpTrigger/Function/PatchInteractionHttpTrigger.cs:1]: _httpResponseMessageHelper and jsonHelper not used any more. These references can be removed

File: NCS.DSS.Interaction/GetInteractionHttpTrigger/Function/GetInteractionHttpTrigger.cs
Comment (PradeepVoorukonda) [inline NCS.DSS.Interaction/GetInteractionHttpTrigger/Function/GetInteractionHttpTrigger.cs:1]: _httpResponseMessageHelper and jsonHelper not used any more. These references can be removed

**Review by PradeepVoorukonda — CHANGES_REQUESTED** [review]
few suggestions / comments

File: NCS.DSS.Interaction/APIDefinition/APIDefinition.cs
[the remark this answers is not in the document]
Comment (amjones99) [reply NCS.DSS.Interaction/APIDefinition/APIDefinition.cs:1]: yes that is correct. It was incorrectly set to 2 before before

**Review by PradeepVoorukonda — APPROVED** [review]

**Pull request merged**